spec/models/post_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(title: 'This is my title', comments_counter: 2, likes_counter: 0) }
+
+  before { subject.save }
+
+  it 'The title should be present' do
+    subject.title.nil?
+    expect(subject).to_not be_valid
+  end
+
+  it 'The title should not exceed 250 characters' do
+    subject.title = 'This is my title' * 30
+    expect(subject).to_not be_valid
+  end
+
+  it 'The comments_counter should be and integer not less than zero' do
+    subject.comments_counter = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'The likes_counter should be an integer not less than zero' do
+    subject.likes_counter = -1
+    expect(subject).to_not be_valid
+  end
+end
```

Comment:
- [ ] You have added tests for the models but they all should be passed. Kindly work on it and make pass all the tests. 
If the tests pass on your side, you can add a screenshot and ask for the next review. :+1: 

![failed_example](https://user-images.githubusercontent.com/70483835/228377078-77c4a9cf-e2b5-499b-9223-26547c8ea180.png)

Reply:
***Hello @mahbubul14***

The tests are passing on my side and I have attached the screenshot 👇👇

![Test screenshot](https://user-images.githubusercontent.com/86472119/228437229-0b0a1304-da88-492c-a746-e66ebbe38b65.png)